indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_handler.py

```diff
@@ -46,8 +46,8 @@ def get_result(self, request: Request):
     def get_current_revoc_entry_and_revoc_def(self, author_did, revoc_reg_def_id, req_id):
```

Comment:
Why do we need `get_current_revoc_entry_and_revoc_def` and `_get_revoc_def_entry` here at all?

Reply:
They used only in RevocRegEntryHandler, but i think logically they need to be defined in GetRevocRegEntry. (Also because they calling lookup function)